src/app/component/SearchField/SearchField.style.scss

```diff
@@ -37,7 +37,6 @@
     }
 
     &_isActive {
-        --search-bar-width: 450px;
```

Comment:
Does not this break tablet? I am not saying it must work on all screen-sizes, but just make sure it look ok on most screen-sizes.

Reply:
Now the value is set to `30vw` so as not to overlap logo when approaching tablet size